README.md

```diff
@@ -4,6 +4,15 @@ The OpenEuropa Starter Content provides a series of content types to quickstart
 
 **Please note:** This project is not compatible with oe_content content types.
 
+#### Drupal 10.1
```

Comment:
remove this as oe_content is a dev dependency

Reply:
Removed